src/main/java/com/hibernate/cinema/dao/MovieSessionDao.java

```diff
@@ -0,0 +1,12 @@
+package com.hibernate.cinema.dao;
+
+import com.hibernate.cinema.model.Movie;
+import com.hibernate.cinema.model.MovieSession;
+import java.time.LocalDate;
+import java.util.List;
+
+public interface MovieSessionDao {
+    MovieSession add(MovieSession movieSession);
+
+    List<MovieSession> getAvailable(Movie movie, LocalDate date);
```

Comment:
Why do you use Movie movie instead movieId ? It makes code more complicated: you have to create additional method, add additional dependency to MovieSessionService...